tests/robottelo/test_decorators.py

```diff
@@ -36,20 +38,49 @@ def test_not_smoke(self):
 class BzBugIsOpenTestCase(TestCase):
     """Tests for :func:`robottelo.decorators.bz_bug_is_open`."""
     # (protected-access) pylint:disable=W0212
-    def setUp(self):  # noqa pylint:disable=C0103
+    # (test names are long to make it readable) pylint:disable=C0103
+    def setUp(self):
         """Back up objects and generate common values."""
         self.backup = decorators._get_bugzilla_bug
         self.bug_id = gen_integer()
+        self.upstream_backup = conf.properties.get('main.upstream')
 
-    def tearDown(self):  # noqa pylint:disable=C0103
+        self.valid_whiteboard_data = [
+            'VERIFIED IN UPSTREAM',
+            'VeRiFiEd In UpStReAm',
+            'verified in upstream'
+        ]
+        self.invalid_whiteboard_data = [
+            'VERIED IN UPSTREAM',
+            'VeRiFiEd I UpStReAm',
+            'verified in uppstream',
+            'None',
+            ''
+        ]
+        self.open_bz_statuses = [
```

Comment:
Good point, I will move statuses to constants as these can be used in decorators.py module as well.  But I will leave valid_whiteboard_data, invalid_whiteboard_data here as these are just test data and wont be used anywhere else.